src/desktop.md

```diff
@@ -321,7 +321,10 @@ and the other DLLs, and bundle them together in a zip file.
 
 #### macOS
 
-The `.app` is self-contained, and can be distributed as-is.
+The `.app` is self-contained, and can be distributed as-is. However, you should 
+read through the [macOS-specific support](#macos-specific-support) section below
+to understand about how Entitlements, the App Sandbox, and the Hardened Runtime
```

Comment:
Nit: Apple capitalizes App Sandbox and Hardened Runtime since they are their own specific terms, but don't capitalize "entitlements" in their docs as it's a more widespread term.